src/wp-content/themes/twentytwentyone/.npmrc

```diff
@@ -1 +0,0 @@
-legacy-peer-deps = true
```

Comment:
Remove this file.

Reply:
Ah, that slipped in accidentally! My bad, I'll remove.

Reply:
5e242ce861